tests/test_email_handler.py

```diff
@@ -61,3 +65,23 @@ def test_is_automatic_out_of_office():
 
     msg[headers.AUTO_SUBMITTED] = "auto-generated"
     assert is_automatic_out_of_office(msg)
+
+
+def test_process_spoofed(flask_client):
```

Comment:
Hope we can have a way to test email handler without side effect soon. Calling `handle` can accidentally call sendmail which requires a SMTP server.

Reply:
Yup. As soon as we have a way to test it independently.